pkg/review/commit_review_status_test.go

```diff
@@ -0,0 +1,454 @@
+// Copyright 2023 The Authors (see AUTHORS file)
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package review
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net/http"
+	"net/http/httptest"
+	"strings"
+	"testing"
+
+	"github.com/abcxyz/pkg/testutil"
+	"github.com/google/go-cmp/cmp"
+	"github.com/shurcooL/githubv4"
+	"golang.org/x/oauth2"
+)
+
+func TestGetPullRequests(t *testing.T) {
+	t.Parallel()
+	cases := []struct {
+		name              string
+		token             string
+		githubOrg         string
+		repository        string
+		commitSha         string
+		responseBodies    []string
+		wantRequestBodies []string
+		want              []*PullRequest
+		wantErr           string
+	}{
+		{
+			name:       "one_pull_request_with_one_page",
+			token:      "ghp_JQqAJky0GlNB7xqVCbUrivIgFZ09V8gGCi5C",
```

Comment:
I'm surprised secret scanner didn't report this. If it's not required, I'd use something more fake like "fake_token"

Reply:
the was just a randomly generated string i used, I can use something that looks more more fake

Reply:
fixed.